airbyte-integrations/bases/standard-destination-test/src/main/resources/v1/data_type_object_test_catalog.json

```diff
@@ -0,0 +1,39 @@
+{
+  "streams": [
+    {
+      "name": "object_test_1",
+      "json_schema": {
+        "type": ["object"],
+        "properties": {
+          "property_string": {
+            "$ref": "WellKnownTypes.json#definitions/String"
+          },
+          "property_date": {
+            "$ref": "WellKnownTypes.json#definitions/Date"
+          },
+          "property_timestamp_with_timezone": {
+            "$ref": "WellKnownTypes.json#definitions/TimestampWithTimezone"
+          },
+          "property_timestamp_without_timezone": {
+            "$ref": "WellKnownTypes.json#definitions/TimestampWithoutTimezone"
```

Comment:
I see we also have TimeWithTimezone and TimeWithoutTimezone. Do we cover it?

Reply:
Adding it. Thanks for noticing